### PR TITLE
Enhancement all movie page

### PR DESCRIPTION
**Pull request by Pyramate**
- edited the movie page to display filter section better

### CODE REVIEW

**Comment by Pyramate** [comment]
[[Enhancement_All_movie_page]](https://trello.com/c/zOTk7Wqw/58-enhancementallmoviepage)

File: frontend/src/pages/AllMovies.jsx [hunk 1]
```diff
@@ -9,17 +10,131 @@ function AllMovies({ sortType }) {
...
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 0 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
```
Comment (lchastanet) [inline frontend/src/pages/AllMovies.jsx:132]: Too many repetitions, there must be a way to manage this a better way 😉

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: frontend/src/pages/AllMovies.jsx (hunk 1 above)
Comment (Pyramate) [reply frontend/src/pages/AllMovies.jsx]: ok this was ugly ^^ but fixed now!

**Review by lchastanet — APPROVED** [review]
Wonderful 😊

**Pull request merged**